src/search/utils/collections.h

```diff
@@ -9,6 +9,7 @@
 #include <unordered_map>
 #include <unordered_set>
 #include <vector>
+#include <iostream>
```

Comment:
Order includes alphabetically, please.

Reply:
I think it is a holdover from debugging code, I will remove it.